src/api/v4/leaflet/layer-group.js

```diff
@@ -84,4 +92,23 @@ LayerGroup.prototype._triggerLayerFeatureEvent = function (eventName, internalEv
   }
 };
 
+function anyLayerHovered (map) {
+  for (var i = 0; i < map.length; i++) {
+    if (map[i]) {
+      return true;
+    }
+  }
+  return false;
+}
+
+/** 
+ * Check if a layer is interactive
+ */
+function isInteractive (layer) {
+  if (!layer) {
+    return false;
+  }
+  return layer.hasFeatureClickColumns() || layer.hasFeatureOverColumns();
```

Comment:
This looks like some [feature envy](https://sourcemaking.com/refactoring/smells/feature-envy)... Why not move this whole method to `carto.layer.Layer` and just do `layer.isInteractive`?